mlflow/tracking/_model_registry/fluent.py

```diff
@@ -701,11 +702,14 @@ def load_prompt(
         name_or_uri: The name of the prompt, or the URI in the format "prompts:/name/version".
         version: The version of the prompt (required when using name, not allowed when using URI).
         allow_missing: If True, return None instead of raising Exception if the specified prompt
-            is not found.
+            is not found. Note: If ``fallback`` is provided, this is automatically set to True.
```

Comment:
I think this will make debugging harder. When I make a typo in the prompt name, I don't want `fallback` to be used.